dashboard/src/components/AppList/AppListItem.tsx

```diff
@@ -75,7 +76,7 @@ function AppListItem(props: IAppListItemProps) {
       }
       description={app.shortDescription}
       tag1Content={appStatus}
-      tag1Class={appStatus === "deployed" ? "label-success" : "label-warning"}
+      tag1Class={appReady ? "label-success" : "label-warning"}
```

Comment:
Squeezing this change in this PR, just for having a pkg-agnostic CSS here as well :P

Reply:
And... squeezing also this other change (https://github.com/kubeapps/kubeapps/pull/3749/commits/cdeead47baf509ed83f27b56bda61a5ebdec3994) for a multi-plugin header subtitles:

![image](https://user-images.githubusercontent.com/11535726/141506818-6b76f853-553f-43b6-bc83-199bbbf3549b.png)

Reply:
That looks much clearer... ah, but this is just on the page header for a package detail? At first I thought you'd added the subtitle on the summary badge. But either way, this is excellent, much clearer.